service/routes.py

```diff
@@ -276,7 +276,7 @@ def delete_items(shopcart_id, item_id):
 
     This endpoint will delete an item based the id specified in the path
     """
-   
+    
```

Comment:
I remember this whitespace would trigger one of the linter error lol

Reply:
Thank you for the reminder! Whitespace removed